.circleci/config.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/diffeo/go-coordinate
+    docker:
+      - image: diffeo/ci-builder:0.1.2-2017122911
+      - image: postgres:9.6
+    steps:
+      - checkout
+
+      - run: git describe HEAD
+
+      - type: setup_remote_docker
+        reusable: true
+
+      - restore_cache:
+          keys:
+            - dep-v1-{{ checksum "Gopkg.lock" }}
+            - dep-v1-
+
+      - run:
+          name: Install dependencies
+          command: |
+            # Ensure that no updates are needed. See:
+            # https://github.com/golang/dep/issues/950
+            dep ensure -v -no-vendor -dry-run
+            # Install dependencies
+            dep ensure -v -vendor-only
```

Comment:
Moving forward it might be nice to have some central script that was "dep ensure, but fail if can't enforce dependencies or stuff was missing", that all the builds used.